cinema_service/settings.py

```diff
@@ -1,39 +1,18 @@
-"""
-Django settings for cinema_service project.
-
-Generated by 'django-admin startproject' using Django 4.0.4.
-
-For more information on this file, see
-https://docs.djangoproject.com/en/4.0/topics/settings/
-
-For the full list of settings and their values, see
-https://docs.djangoproject.com/en/4.0/ref/settings/
-"""
+import os
 from datetime import timedelta
 from pathlib import Path
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
 BASE_DIR = Path(__file__).resolve().parent.parent
 
+SECRET_KEY = os.environ.get("SECRET_KEY",
+                            "django-insecure-6vubhk2$++agnctay_4pxy_8cq)"
+                            "mosmn(*-#2b^v4cgsh-^!i3")
```

Comment:
Using a hardcoded secret key is not recommended for production environments. Ensure that the `SECRET_KEY` is securely set in the environment variables and not exposed in the code.